src/style.css

```diff
@@ -1,3 +1,99 @@
+/* zeroing */
+
+img {
+  vertical-align: top;
+}
+
+a {
+  text-decoration: none;
+}
+
+ul,
+li {
+  list-style: none;
+}
+
+html,
 body {
+  font-family: Roboto, sans-serif;
+  color: #000;
+  font-size: 12px;
   margin: 0;
+  line-height: 1;
+  box-sizing: border-box;
+}
+
+/* -------------------------- wrapper -------------------------- */
+
+.wrapper {
+  min-height: 100%;
+  overflow: hidden;       /* to remove horizontal scroll */
+}
+
+/* -------------------------- container -------------------------- */
+
+[class*="__container"] {
+  max-width: 1230px;
+  margin: 0 auto;
+  padding: 0 15px;       /* padding for mobile devices, so max-width: 1230px */
+}
+
+@media (max-width: 1230px) {
```

Comment:
Media query is redundant in this task. Display flex  with `space-between` property solves all of this